src/metabase/driver.clj

```diff
@@ -127,6 +127,9 @@
           :rows    [[1 \"Lucky Bird\"]
                     [2 \"Rasta Can\"]]}")
 
+  (cancel-query ^java.util.Map [this, ^String query-id]
```

Comment:
Was thinking people would do something like:

    {:status "pending"}

or whatever makes sense to the UI for this when it exists. Not a strong opinion.